### DISCUSSION TITLE
Language support

### DISCUSSION THREAD

**Discussion by rahulagarwal86** [code-shaper/code-shaper#10] [General]
Does it only support FE languages or does it have a support for BE languages too like Python

**Comment by nareshbhatia** [comment]
Hi Rahul, Code Shaper can generate code in any language - you just have to write a plugin that contains templates in that language. Please refer to [Getting Started](https://www.code-shaper.dev/docs/getting-started/overview) to learn how to write your own plugins and generators. You may also like to read about the [plugin architecture](https://www.code-shaper.dev/docs/concepts/plugin-architecture).

If you write a plugin for Python, would love to add it to the [Code Shaper Plugin Directory](https://www.code-shaper.dev/docs/community/code-shaper-plugin-directory)
